src/Search/Console.php

```diff
@@ -120,6 +120,10 @@ public function __construct(Application $console, Kernel $app)
         $this->app = $app;
         $this->root = __DIR__.'/../..';
 
+        if (!defined('GEARMAN_INSTALLED')) {
```

Comment:
this should never be false. Where it is required? Should be made explicit in the composer.json as `ext-gearman`